examples/getstarted/src/policies/test-policy.js

```diff
@@ -1,3 +1,18 @@
-module.exports = ctx => {
-  return true;
+'use strict';
+
+/**
+ * `coucou-policy` policy.
```

Comment:
Maybe replace `coucou` by `test` to match the filename?

Reply:
👍